selfdrive/ui/paint.cc

```diff
@@ -127,25 +127,23 @@ static void draw_lead(UIState *s, const cereal::RadarState::LeadData::Reader &le
   draw_chevron(s, d_rel, lead.getYRel(), 25, nvgRGBA(201, 34, 49, fillAlpha), COLOR_YELLOW);
 }
 
-static void ui_draw_lane_line(UIState *s, const model_path_vertices_data *pvd, NVGcolor color) {
+static void ui_draw_lane_line(UIState *s, const model_path_vertices_data *pvd, NVGcolor color, bool is_ghost) {
```

Comment:
You were able to remove lines by assuming the order of lane line drawing?

Reply:
If that's true, I'm not a fan of this change. Sounds like a time-consuming bug waiting to be written the next time someone touches this code.

Reply:
nothing about the order of drawing.
the ghost line's width is always 1px.we don't need to draw it twice(draw back from line end to start), that can save 196 calls to car_space_to_full_frame and  lineto if path.validLen is 98